recipes/wlogdate/meta.yaml

```diff
@@ -1,25 +1,32 @@
-{% set version = "1.0.0" %}
+{% set version = "1.0.3" %}
 
 package:
   name: wlogdate
   version: {{ version }}
 
 source:
   url: https://github.com/uym2/wLogDate/archive/v{{ version }}.tar.gz
-  sha256: dc0beed7554bf35354c2c6fb4330cf73df92020f46317a6caa5875af05a0bf25
+  sha256: 1428cb82f2c8151c9711efcb4f916fa9951bbd2ff8a9f910a4b76dc612c8ece0
 
 build:
-  number: 0
+  number: 1
```

Comment:
This should be reset to 0 ...

Reply:
Just fixed. Thanks!

Reply:
I now get the error:
> The recipe build number should be incremented', body='\nA package with the same name and version and a build number at\nleast as high as specified in the recipe already exists in the\nchannel. Please increase the build number.', start_line=11, end_line=12, fname='recipes/wlogdate/meta.yaml', canfix=True)

I am guessing because it worked fine with 1 once now, it is not able to back to 0 now? Should I go back to 1 or 2?

Reply:
Wait. Sorry, I think the problem was that this version was already updated on bioconda (perhaps using a bot) and I hadn't realized. So I Should remove this pull request. It is not necessary.